src/System.Private.CoreLib/shared/System/Span.Fast.cs

```diff
@@ -159,6 +159,24 @@ internal Span(ref T ptr, int length)
 #endif
         }
 
+        public ref readonly T this[Index index]
```

Comment:
Why readonly?

Reply:
you are right. I'll keep the indexer consistent with this[int] in both Span and ReadOnlySpan. I mean would be readonly in ReadOnlySpan